src/components/views/auth/InteractiveAuthEntryComponents.tsx

```diff
@@ -657,7 +691,7 @@ export class SSOAuthEntry extends React.Component {
         // certainly will need to open the thing in a new tab to avoid losing application
         // context.
 
-        this._popupWindow = window.open(this._ssoUrl, "_blank");
+        this.popupWindow = window.open(this.ssoUrl, "_blank");
         this.setState({phase: SSOAuthEntry.PHASE_POSTAUTH});
         this.props.onPhaseChange(SSOAuthEntry.PHASE_POSTAUTH);
     };
```

Comment:
github diff won't let me get at it, but on new-line 699 `onConfirmClick` needs explicit member access